test/test_ops.py

```diff
@@ -463,6 +439,115 @@ def _fn(*sample_args, **sample_kwargs):
                 graph = traced.graph_for(*inp)
                 FileCheck().check(op_name).check_not(variant_name).run(graph)
 
+    # Validates ops implement the correct out= behavior
+    # See https://github.com/pytorch/pytorch/wiki/Developer-FAQ#how-does-out-work-in-pytorch
+    #   for a description of the correct behavior
+    @ops(op_db, dtypes=OpDTypes.supported, allowed_dtypes=(torch.float,))
```

Comment:
How is dtypes different from allowed_dtypes?

Reply:
dtypes overrides the usual op info dtype generation; allowed_dtypes intersects with the given dtypes so only those are run. In this case just using allowed_dtypes is correct.